dcos/nodeutil/util.go

```diff
@@ -30,7 +31,7 @@ type NodeInfo interface {
 	DetectIP() (net.IP, error)
 	Role() (string, error)
 	IsLeader() (bool, error)
-	MesosID() (string, error)
+	MesosID(*context.Context) (string, error)
```

Comment:
today i learned. I had impression context.Context is a structure not an interface. Thanks for a good catch.